### ISSUE TITLE
submitting subnetworks to patch up existing AlchemicalNetworks with gaps doesn't work

### ISSUE THREAD

**Issue by JenkeScheen** [asapdiscovery/asapdiscovery#1174]
I have an `AlchemicalNetwork` that has a consistently failing edge in its MST topology which causes it to be disconnected. The obvious approach to fix this is by adding a bunch of new edges (as a new network) and then submitting them to the `AlchemicalNetwork`; currently for some reason when I do this `asap-alchemy submit` doesn't submit the new subnetwork but instead just submits the original larger `AlchemicalNetwork` again.

**Comment by hmacdope** [comment]
@JenkeScheen did we get to the bottom of this in the end?

**Comment by JenkeScheen** [comment]
I don't remember - @dotsdl do you? If @dotsdl doesn't know then this is still on to-do

**Comment by dotsdl** [comment]
@JenkeScheen `AlchemicalNetwork`s are submitted to `Scope`s, not to other `AlchemicalNetwork`s, so I'm a bit confused by the description. If two `AlchemicalNetwork`s in the same `Scope` share some `Transformation`s (and therefore `ChemicalSystem`s), they will share the results for those `Transformation`s. However, submitting a smaller `AlchemicalNetwork` with `Transformation`s spanning some `ChemicalSystem`s present in an existing larger `AlchemicalNetwork` will not "add" those `Transformation`s to the larger `AlchemicalNetwork`.

In other words, `AlchemicalNetwork`s are immutable, but you can "grow" them by submitting `AlchemicalNetwork`s that are supersets of previous ones to the same `Scope`.
